### PR TITLE
Color right click menu only in Player Indicators

### PR DESCRIPTION
**Pull request by Tunnelsnake52**
A toggle to only color in the right-click menu and not to have names drawn overhead

Toggle Off
image

Toggle On
image

### CODE REVIEW

**Comment by devLotto** [comment]
This closes #1523

File: runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsConfig.java [hunk 1]
```diff
@@ -180,7 +180,18 @@ default boolean colorPlayerMenu()
 	}
 
 	@ConfigItem(
-		position = 13,
```
Comment (devLotto) [inline runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsConfig.java]: Don't change indentation

**Comment by devLotto** [comment]
Perhaps an option "colorWorld" or something that is enabled by default should be added instead of "only rightclick menu". 
What do you think @deathbeam ?

File: runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsService.java [hunk 2]
```diff
@@ -70,22 +70,26 @@ public void forEachPlayer(final BiConsumer<Player, Color> consumer)
 					consumer.accept(player, config.getOwnNameColor());
 				}
 			}
-			else if (config.drawFriendNames() && player.isFriend())
+			if (!config.onlyPlayerMenu())
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsService.java]: This logic should probably also be placed near whatever draws player names, and not here which just allows parts of the plugin to pass in a consumer.

**Comment by sethtroll** [comment]
Added in #2012

**Pull request closed without merging**